.github/workflows/deploy-happy-stack.yml

```diff
@@ -94,16 +94,17 @@ jobs:
         if: failure() && github.ref == 'refs/heads/main'
 
   smoke-test:
```

Comment:
if we're renaming the Run Smoke Tests -> Run E2E tests below, maybe we can rename this overall job name to E2E tests too?